commonknowledge/mapped
# Default to 2024 constituencies + analytical area toggler

- Defaults private report maps to 2024 constituencies
- Adds new GraphQL fields for analytical queries on any area type
- Adds a dropdown UI for toggling between constituency area types

## Motivation and Context

The old constituencies are out, so we want people focused on 2024 for the campaign.

Moving forwards, post-election, there will be other more pertinent geographies. This PR adds preliminary code to run analytical queries on any kind of area.

## How Can It Be Tested?

1. Load a new map. Constituencies should be 2024 by default.
2. Toggle the map (Map Layers -> Westminster Politics) and the map should change and the numbers should change.

## How Will This Be Deployed?

N/A

## Screenshots (if appropriate):

<img width="996" alt="Screenshot 2024-06-06 at 13 29 14" src="https://github.com/commonknowledge/mapped/assets/237556/ade50873-d676-4bcf-bceb-8f150b6e1b68">
<img width="1035" alt="Screenshot 2024-06-06 at 13 29 17" src="https://github.com/commonknowledge/mapped/assets/237556/035bdb7e-5d1b-4397-b0ec-394a0a0a6a41">

## Types of changes
